### DISCUSSION TITLE
Not sure if this is a bug: scoop update * vs scoop update

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [ScoopInstaller/Scoop#6250] [Help and Questions]
I've noticed that immediately after invoking:
```
scoop update *
```
if you then invoke:
```
scoop update
```
Scoop will sometimes find app manifests to update.  Shouldn't `scoop update *` always update all buckets to the have the latest manifests for every app?

If so, is this a bug?  If not, why does `scoop update` sometimes find manifests to update that `scoop update *` does not?

**Comment by niheaven** [comment]
Very simple, manifest repos just update these apps during `scoop update *` running.

**Reply by Gitoffthelawn** [reply]
> Very simple, manifest repos just update these apps during `scoop update *` running.

I'm not understanding what you are trying to convey.  When you state "these apps", to which apps are you referring?

When I invoke `scoop update *`, I see a list of many apps, not just those installed.

**Reply by niheaven** [reply]
`scoop update` updates scoop itself and the buckets (i.e. manifests), while `scoop update *` first calls `scoop update`, and then updates any outdated apps if they exist.

Additionally, `scoop update *` will run `scoop update` only if it has been 4 hours since the last run. During this 4-hour period, `scoop update *` will not run `coop update` again, even if there are bucket updates available online.
